### PR TITLE
Add Elkjøp post

### PR DESCRIPTION
**Pull request by cnunciato**

### CODE REVIEW

File: themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+date: 2022-06-23
+meta_desc: How Elkjøp Nordic enables developers to self-serve infrastructure
+meta_image: meta.png
+authors:
+    - george-huang
+tags:
+    - change-me
+---
+
+At PulumiUP 2022, Tomas Jansson, software architect at Elkjøp Nordic, gave a presentation on how to enable developers to self-service infrastructure by using Pulumi’s Automation API.
```
Comment (gthuang) [inline themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md:13]: link to Automation API marketing page

**Comment by gthuang** [comment]
please add an entry to https://www.pulumi.com/case-studies/

To support the company’s modernization strategy, Elkjøp developed a self-service infrastructure platform that would enable its developers to easily provision cloud infrastructure that uses approved resources and security practices. The goal was to empower teams at Elkjøp to increase their development velocity by using the cloud, while maintaining guardrails that uphold security and compliance.

File: themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+title: How Elkjøp Nordic enables self-service infrastructure for developers
+allow_long_title: true
+date: 2022-06-23
+meta_desc: How Elkjøp Nordic enables developers to self-serve infrastructure
```
Comment (gthuang) [inline themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md:5]:
```suggestion
meta_desc: Learn how Elkjøp Nordic enables its developers to self-service Azure infrastructure with compliance guardrails using Pulumi infrastructure as code.
```

File: themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+![Deploying with the Pulumi CLI and Automation API](./automation-api-diagram.png)
+
+The Pulumi Automation API is a programmatic interface for running Pulumi programs without the Pulumi CLI. Automation API encapsulates the functionality of the CLI (`pulumi up`, `pulumi preview`, `pulumi destroy`, `pulumi stack init`, etc.) but with more flexibility since infrastructure operations can be controlled programmatically.
+
+## How the self-service infrastructure platform was built
+
+Here is the workflow for the Elkjøp self-service infrastructure platform.
+
+![The Elkjøp self-service infrastructure platform](./elkjop-platform-diagram.png)
```
[the remark this answers is not in the document]
Comment (cnunciato) [reply themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md:56]: It's not quite the right shape, as those images are rectangular. 🤔

File: themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## Watch the demo
+
+To illustrate how Elkjøp’s self-service platform works, Tomas presented a simplified demo application that’s based on Elkjøp’s production platform. The code for the application is available at <https://github.com/mastoj/pulumi-automation-demo>.
+
+The demo is a TypeScript application that includes an API that provisions Azure and GitHub resources. Pulumi Automation API code is used to deploy new stacks by calling handler code, which is a Pulumi program that defines resources and outputs. The handler code can create Azure resource groups or GitHub repositories, and stack references are used so that outputs generated by the Azure resources can be stored in GitHub (e.g., resource group credentials).
+
+Ready to dive into how this works in detail? Watch the demo here:
+
+{{< youtube "aoa_O-rh5KE?t=620" >}}
```
[the remark this answers is not in the document]
Comment (cnunciato) [reply themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md:72]: I think maybe Hugo doesn't support the time marker attribute. I'll remove, and then if we need to, we can just say where to skip to.

File: themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Elkjøp Nordic is the leading consumer electronics retailer in the Nordics. The company sells consumer electronics, mobile phones, computers, white goods, domestic appliances, and services linked to these products both directly to consumers and to businesses. It is an omnichannel retailer and serves customers both online and through more than 400 stores. Elkjøp has retail outlets in Norway, Sweden, Finland and Denmark, and franchise operations in Greenland and Faroe Islands.
+
+To support the company’s modernization strategy, Tomas developed a self-service infrastructure platform that would enable any developer to easily provision cloud infrastructure that uses approved resources and security practices. The goal was to empower teams at Elkjøp to increase their development velocity by using the cloud, while maintaining guardrails that uphold security and compliance.
+
+Watch the full presentation below or read on to learn the highlights of his presentation:
+
+{{< youtube "https://youtu.be/aoa_O-rh5KE" >}}
```
Comment (cnunciato) [inline themes/default/content/blog/how-elkjop-nordic-enables-developers-to-self-serve-infrastructure/index.md:30]:
```suggestion
{{< youtube "aoa_O-rh5KE" >}}
```

**Review by gthuang — APPROVED** [review]

**Comment by susanev** [comment]
@gthuang @cnunciato the case studies list page orders them in alphabetic order, can yall pls maintain that with this new addition?

**Pull request merged**